breacharbiter_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !rpctest
```

Comment:
I thought this flag was for integration tests?

Reply:
Correct! This one is saying to _not_ include the unit tests in this file when compiling the integration tests, note the exclaimation point :)

Reply:
ah, right XD